client/Windows/wfreerdp_mingw.rc

```diff
@@ -0,0 +1,14 @@
+

```

Comment:
This is exactly the same as `wfreerdp.rc`

Reply:
The content is same, but encoding is different. windres requires .rc files to be UTF-8, while MS suite uses .rc files encoded as UTF16+BOM. This is the only difference between those files.

Reply:
the `UTF16` might actually not be required